docs/book/src/user/quick-start.md

```diff
@@ -0,0 +1,152 @@
+# Quick Start
+
+In this tutorial we'll cover the basics of how to use Cluster API to create one or more Kubernetes clusters.
+
+## Prerequisites
+
+- Install and setup [kubectl] in your local environment.
+- A Kubernetes cluster to be used as [management cluster](../reference/glossary.md#management-cluster), for the purpose of this quick start, we suggest you to use [kind].
+    ```bash
+    kind create cluster --name=clusterapi
+    export KUBECONFIG="$(kind get kubeconfig-path --name="clusterapi")"
+    ```
+
+## Installation
+
+Using [kubectl], let's create the components on the management cluster:
+
+#### Install Cluster API Components
+```bash
+kubectl create -f https://github.com/kubernetes-sigs/cluster-api/releases/latest/download/cluster-api-components.yaml
+```
+
+#### Install the Bootstrap Provider Components
+
+{{#tabs name:"tab-installation-bootstrap" tabs:"Kubeadm"}}
+{{#tab Kubeadm}}
+
+```bash
+kubectl create -f https://github.com/kubernetes-sigs/cluster-api-bootstrap-provider-kubeadm/releases/latest/download/bootstrap-components.yaml
+```
+
+{{#/tab }}
+{{#/tabs }}
+
+
+#### Install Infrastructure Provider Components
+
+{{#tabs name:"tab-installation-infrastructure" tabs:"AWS"}}
+{{#tab AWS}}
+
+For more information about prerequisites, credentials management, and or permissions for AWS, visit the [getting started guide](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/docs/getting-started.md).
+
+```bash
+kubectl create -f https://github.com/kubernetes-sigs/cluster-api-bootstrap-provider-kubeadm/releases/latest/download/infrastructure-components.yaml
+```
+
+{{#/tab }}
+{{#/tabs }}
+
+## Usage
+
+The Cluster API resources are now installed.
+
+![](../images/management-cluster.svg)
+
+Now that we've got Cluster API, Bootstrap and Infrastructure resources installed,
+let's proceed to create a single node cluster.
+
+For the purpose of this tutorial, we'll name our cluster `capi-quickstart`.
+
+{{#tabs name:"tab-usage-cluster-resource" tabs:"AWS"}}
+{{#tab AWS}}
+
+```yaml
+---
+apiVersion: cluster.x-k8s.io/v1alpha2
+kind: Cluster
+metadata:
+  name: capi-quickstart
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks: ["192.168.0.0/16"]
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+    kind: AWSCluster
+    name: capi-quickstart
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+kind: AWSCluster
+metadata:
+  name: capi-quickstart
+spec:
+  region:
+  sshKeyName: default
+```
+{{#/tab }}
+{{#/tabs }}
+
+Now that we've created the cluster object, we can create the first control plane Machine.
```

Comment:
If we are only going to deploy a single control plane instance for the quick start (which we likely should):
```suggestion
Now that we've created the cluster object, we can create a control plane Machine.
```